src/repairchain/strategies/generation/llm/yolo_llm.py

```diff
@@ -401,20 +402,31 @@ def _get_patches_per_file(self, code_summary: list[FunctionSummary] | None) -> t
         if self.diagnosis.implicated_functions_at_head is None:
             return
 
+        logger.debug(f"functions to consider: {self.diagnosis.implicated_functions_at_head}")
```

Comment:
this is a LOT to log. this might be the problem. I suggest just reporting the names instead.